.github/workflows/carpenter_build.yaml

```diff
@@ -1,14 +1,12 @@
 name: Carpentry
 on:
-  push:
-    branches:
-      - main
   pull_request:
 env:
-  IMAGE_NAME: arcaflow-plugin-elasticsearch
-  IMAGE_TAG: 'latest'
+  IMAGE_NAME: arcaflow-plugin-uperf
```

Comment:
Why was this change merged?  This appears to be in the `elasticsearch` plugin repository, no?

Reply:
Looks like this was fixed by #5.